sheeprl/configs/exp/dreamer_v3.yaml

```diff
@@ -8,6 +8,7 @@ defaults:
 
 # Algorithm
 algo:
+  train_after_step: true
```

Comment:
Here you set an argument with a different name than the other file, I ask you to change its name and default value according to the previous comment and move the definition of this parameter in the `./sheeprl/configs/algo/dreamer_v3.yaml` file.
Please add also a brief comment where you indicate the meaning of the parameter.